Portfolio-Review-Service/src/test/java/PortfolioReviewService/repo/CriteriaRepoTest.java

```diff
@@ -0,0 +1,47 @@
+package PortfolioReviewService.repo;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.when;
+
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import com.forge.PortfolioReviewService.models.Criteria;
+import com.forge.PortfolioReviewService.repository.CriteriaRepo;
+
+@SpringBootTest(classes=Criteria.class)
+public class CriteriaRepoTest {
+	
+	@MockBean
+	private CriteriaRepo cRepo;
+	
+	@Test
```

Comment:
These tests aren't actually testing anything, they literally cannot fail. You're mocking the methods you test. That being said, there's no need to test jpa repositories. If you wanted to do some thing similar, you could mock the repo in the service tests and verify the number of times certain methods from the repo were called.